lib/editor/components/EntityList.js

```diff
@@ -8,29 +8,34 @@ import {AutoSizer} from 'react-virtualized/dist/commonjs/AutoSizer'
 
 import * as activeActions from '../actions/active'
 import * as editorActions from '../actions/editor'
+import * as snapshotActions from '../actions/snapshots'
 import {getConfigProperty} from '../../common/util/config'
 import {componentToText, entityIsNew} from '../util/objects'
 import EntityListButtons from './EntityListButtons'
 import EntityListSecondaryActions from './EntityListSecondaryActions'
 
 import type {Props as ContainerProps} from '../containers/ActiveEntityList'
 import type {Entity, Feed} from '../../types'
-import type {DataStateSort} from '../../types/reducers'
+import type {DataStateSort, EditorTables} from '../../types/reducers'
 import type {ImmutableList} from '../selectors/index'
 
 type Props = ContainerProps & {
   activeEntity: Entity,
+  approveGtfsDisabled: boolean,
   cloneGtfsEntity: typeof editorActions.cloneGtfsEntity,
+  createSnapshot: typeof snapshotActions.createSnapshot,
   deleteGtfsEntity: typeof activeActions.deleteGtfsEntity,
   enterTimetableEditor: typeof activeActions.enterTimetableEditor,
   entities: Array<Entity>,
   feedSource: Feed,
   hasRoutes: boolean,
   list: ImmutableList, // FIXME Immutable.List
   newGtfsEntity: typeof editorActions.newGtfsEntity,
+  patchTable: typeof editorActions.patchTable,
   resetActiveGtfsEntity: typeof activeActions.resetActiveGtfsEntity,
   setActiveEntity: typeof activeActions.setActiveEntity,
   sort: DataStateSort,
+  tableData: EditorTables,
```

Comment:
Lots and lots of props here and they’re all passed to the `EntityListButtons` and `EntityListSecondaryActions` components. A new container or two could be created to not require passing through so many props.